internal/proto/controller/api/services/v1/scope_service.proto

```diff
@@ -133,3 +181,38 @@ message DeleteScopeRequest {
 }
 
 message DeleteScopeResponse {}
+
+message ListKeysRequest {
+  string id = 1;
+}
+
+message ListKeysResponse {
+  repeated resources.scopes.v1.Key items = 1;
+}
+
+message RotateKeysRequest {
+  string scope_id = 1;
+  bool rewrap = 2;
+}
+
+message RotateKeysResponse {}
+
+message ListKeyVersionDestructionJobsRequest {
+  string scope_id = 1;
+}
+
+message ListKeyVersionDestructionJobsResponse {
+  repeated resources.scopes.v1.KeyVersionDestructionJob items = 1;
+}
+
+message DestroyKeyVersionRequest {
+  string scope_id = 1;
+  string key_version_id = 2;
+}
+
+message DestroyKeyVersionResponse {
+  // Destruction state. One of "pending" or "completed".
+  // Use GET /v1/scopes/{scope_id}:list-key-version-destruction-jobs
+  // to monitor pending destruction jobs.
+  string state = 1;
```

Comment:
This state describes the destruction operation the user requested. It's closely related to the job status, but not identical. This `state` will be `completed` for any destructions that could be executed immediately (a root key, or a data key with no data to rewrap). The state will be `pending` for any destructions that necessitated the creation of a destruction job. We considered making this a bool since it only has two values, but considered this to be a clearer way to convey to the user the result of their operation. If you would prefer a bool, what would we name it?

Reply:
This is fine.  Ty for the explanation.